graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -108,107 +135,139 @@ public void upgrade() {
+                "/usr/lib/graylog-sidecar/auditbeat",
+                "-c  %s",
+                "test config -c %s",
+                f("""
+                                %s
+                                auditbeat.modules:
+                                - module: file_integrity
+                                  paths:
+                                  - /bin
+                                  - /usr/bin
+                                  - /sbin
+                                  - /usr/sbin
+                                  - /etc
+                                output.logstash:
+                                   hosts: ["${user.graylog_host}:5044"]
+                                path:
+                                  data: /var/lib/graylog-sidecar/collectors/auditbeat/data
+                                  logs: /var/lib/graylog-sidecar/collectors/auditbeat/log""",
+                        beatsPreambel)
+        ).ifPresent(collector -> ensureDefaultConfiguration("default-linux", collector));
         ensureCollector(
                 "winlogbeat",
                 "svc",
                 "windows",
                 "C:\\Program Files\\Graylog\\sidecar\\winlogbeat.exe",
                 "-c \"%s\"",
                 "test config -c \"%s\"",
-                beatsPreambel +
-                        "output.logstash:\n" +
-                        "   hosts: [\"192.168.1.1:5044\"]\n" +
-                        "path:\n" +
-                        "  data: ${sidecar.spoolDir!\"C:\\\\Program Files\\\\Graylog\\\\sidecar\\\\cache\\\\winlogbeat\"}\\data\n" +
-                        "  logs: ${sidecar.spoolDir!\"C:\\\\Program Files\\\\Graylog\\\\sidecar\"}\\logs\n" +
-                        "tags:\n" +
-                        " - windows\n" +
-                        "winlogbeat:\n" +
-                        "  event_logs:\n" +
-                        "   - name: Application\n" +
-                        "   - name: System\n" +
-                        "   - name: Security"
-        );
+                f("""
+                                %s
+                                output.logstash:
+                                   hosts: ["${user.graylog_host}:5044"]
+                                path:
+                                  data: ${sidecar.spoolDir!"C:\\\\Program Files\\\\Graylog\\\\sidecar\\\\cache\\\\winlogbeat"}\\data
+                                  logs: ${sidecar.spoolDir!"C:\\\\Program Files\\\\Graylog\\\\sidecar"}\\logs
+                                tags:
+                                 - windows
+                                winlogbeat:
+                                  event_logs:
+                                   - name: Application
+                                   - name: System
+                                   - name: Security""",
+                        beatsPreambel
+                )
+        ).ifPresent(collector -> ensureDefaultConfiguration("default-windows", collector));
```

Comment:
```suggestion
        ).ifPresent(collector -> ensureDefaultConfiguration("winlogbeat-default", collector));
```
I'd rather name that after the collector. the OS is implicit. actually the collector is too, but I think it looks better this way